src/wolfgame.py

```diff
@@ -7432,15 +7432,15 @@ def pyeval(cli, nick, chan, rest):
             else:
                 cli.msg(chan, a[:500])
         except Exception as e:
-            cli.msg(chan, str(type(e))+":"+str(e))
+            cli.msg(chan, repr(e))
```

Comment:
It doesn't. I just did it alongside when I was testing. I figured it was so minor that it didn't matter, but if it does I'll just push it directly.